model/versioning/Versioned.php

```diff
@@ -1556,16 +1647,7 @@ public static function set_reading_mode($mode) {
 	 * @return string
 	 */
 	public static function get_reading_mode() {
-		return Versioned::$reading_mode;
-	}
-
-	/**
-	 * Get the name of the 'live' stage.
-	 *
-	 * @return string
-	 */
-	public static function get_live_stage() {
```

Comment:
Shouldn’t we deprecate this? I know we _can_ just remove it in a new major version, but it’s probably kinder to raise a dep notice for now. Saves devs having to refer back to the upgrade docs when they get a fatal error for a method not existing. Same for `current_stage()` and `reading_stage()`